faculty-affiliations.csv

```diff
@@ -3357,6 +3359,7 @@ Gregory Dudek , McGill University
 Hamed Hatami , McGill University
 Jackie Chi Kit Cheung , McGill University
 Jérôme Waldispühl , McGill University
+Joelle Pineau , McGill University
```

Comment:
Does not meet inclusion criteria as will be full-time employed elsewhere.

Reply:
Fine ..